NOSQL/Assignment_doument_structure.txt

```diff
@@ -0,0 +1,37 @@
+Users
+{
+	id:1
+	user_name:"abc",
+	password:"zbxc123"
+	name:"anc asd",
+	date_of_birth:12-12-1990,
+	gender:"m or f"
+}
+
+Posts
+{
+	author_id:1
```

Comment:
preferably use the collection's name as a prefix to the reference key "user_id"  instead of "author_id".